webapp/channels/src/components/get_link_modal.tsx

```diff
@@ -69,18 +76,12 @@ export default class GetLinkModal extends React.PureComponent<Props, State> {
 
         if (document.queryCommandSupported('copy')) {
             copyLink = (
-                <button
-                    id='linkModalCopyLink'
-                    data-copy-btn='true'
```

Comment:
@hmhealey This is something lost during the migration. It doesn't seem to be used anywhere, and I couldn't find documentation on whether it does anything by default, but just in case I mention you.

Reply:
I think that probably dates back to either when we used Bootstrap's JS code or whatever copying library we were using at the time. You're good to remove it